x-pack/plugins/cases/server/services/attachments/operations/get.ts

```diff
@@ -59,7 +50,7 @@ export class AttachmentGetter {
       );
 
       const response =
-        await this.context.unsecuredSavedObjectsClient.bulkGet<AttachmentAttributesWithoutRefs>(
+        await this.context.unsecuredSavedObjectsClient.bulkGet<AttachmentPersistedAttributes>(
```

Comment:
Same here, I think we should add the new type to all SO methods of the `AttachmentGetter` class.

Reply:
I think the ones I didn't add it to here are intentionally wanting `unknown`. Let me know if I missed any though.